### ISSUE TITLE
There's no easy way to return to finite sand.

### ISSUE THREAD

**Issue by Prezombie** [eternaldensity/Sandcastle-Builder#503]
As it is, once you buy badgers, mushrooms, or purifier and extender reach infinity, it's impossible to make your sand finite. Even buying crates doesn't work, the 1 mNP after, sand is mustard, not 0. The only way is to reach mustard tools then nuke the lot at once.

Problem is, without finite sand, it a stupid chore to reach inf tools, because the glassed lightning needs to run out each time, and if you are unlucky to not have the lightning rod already, Thunderbird causes Lightning to _lower_ tool production, going to 72 GW from 144 GW max.

To summarize, if Infsand - infsand = Mustard, then Mustard sand allow blitzing. That, with a way to produce Mustard Sand/mNP, perhaps as a boost unlocked when you click on the mustarded Newpix (which deserve a discovery or two) would be a reasonable late-game addition so one doesn't need to rely on an unbadgered and unmushroomed glass infrastructure.
